### ISSUE TITLE
Unable to take prod build

### ISSUE THREAD

**Issue by ghost** [ibm-wch/sample-sites-pzn#2]
When I try to take a run build and live serve. I am getting the following error.

<img width="703" alt="screen shot 2018-10-31 at 12 09 00 am" src="https://user-images.githubusercontent.com/26321221/47741802-37dee500-dca1-11e8-85fd-c01fe1d7c854.png">

**Comment by ThomasHurek** [comment]
@tangentkitty Can you please paste your tenant ID?

**Comment by mburati** [comment]
@tangentkitty - please also verify that the npm run build-deploy worked successfully, and that you edited src/app/Constants.ts and uncommented and set the content hub id and hostname for your tenant correctly for your tenant?

**Comment by ghost** [comment]
@mburati @ThomasHurek 
Please find the following,
  static readonly DOMAIN_NAME = 'my11.digitalexperience.ibm.com';
  static readonly CONTENT_HUB_ID = '5c7faf90-6089-427d-8eb7-14afb7f29a8a';

**Comment by ThomasHurek** [comment]
@tangentkitty I tried accessing your site via https://my11.digitalexperience.ibm.com/5c7faf90-6089-427d-8eb7-14afb7f29a8a/home and that seemed to work. So I am assuming just the localhost display does not work when you access http://127.0.0.1:8080 ?

**Comment by ghost** [comment]
@ThomasHurek Yes when I give npm start it runs without an issue. However when I take prod build. I am getting those errors.
